sphinx_doc/migration/Eloquent.rst

```diff
@@ -42,3 +42,17 @@ Many new behavior tree nodes were added.
 These behavior tree nodes are now BT plugins and dynamically loadable at run-time using behavior tree cpp v3.
 See ``nav2_behavior_tree`` for a full listing, or :ref:`plugins` for the current list of behavior tree plugins and their descriptions. 
 These plugins are set as default in the ``nav2_bt_navigator`` but may be overrided by the ``bt_plugins`` parameter to include your specific plugins.
+
+Map Server Re-Work
+******************
+
+Map Server now has new MapIO dynamic library. All map input-output functionality was moved from ``map_server`` and ``map_saver`` here. MapIO contains loading/saving functions working with ``OccupancyGrid`` messages: ``loadMapYaml()``, ``loadMapFromFile()``, ``loadMapFromYaml()`` and ``saveMapToFile()``. These functions could be easily called from any part of external ROS2 code even if Map Server node was not started.
```

Comment:
You can delete lines 49 and 50, this should be a short overview.

Reply:
I've removed these lines in update patch. But still not sure about not telling about MapIO. It is a kind of new library designed to be used from outside with `map_io.hpp` function calls. Why do you think it should be skipped?

Reply:
Lets include it. I was trying to keep it short but if you think its good context, lets do it. I’d ask that we make API level changes first (the new map saver server + refector) in the list.

Reply:
I've got it. `map_io` description was shortened and moved to the bottom of "Map Server Re-Work" chapter. Added back in 08a9cc1 commit. Hope it is OK now.